napari-workshops/intro_napari_GUI.md

```diff
@@ -0,0 +1,209 @@
+        ```
+
+## Explore Layer Types  
+
+The Points, Shapes, and Labels layers can be used for annotating you image layers and are accessible 
+through the GUI buttons. Additional data layers (Surfaces, Tracks, and Vectors) are available through 
+the API. For guides to using the various, please see the [napari layer guides](https://napari.org/stable/howtos/layers/index.html).
+
+* Add a points, shapes, or labels layer using the buttons above the layers list.
+    * These layers will be blank to begin with.  
+    * In 2D mode, use the layer controls to add points, draw some shapes, or paint some labels on your image (referring to points, shapes, and labels layers respectively).
+* Re-order the layer list by dragging a selected layer in the list below another. Depending on the blending and opacity settings, this will change the visualization!
+
+```{tip}
+You can delete layers selected individually using the trash can icon or by pressing the backspace key.
+Delete all layers in the layer list by selecting one layer, and then pressing Command+A(Mac) / Control+A(Windows) to select all layers. Then click the trash can button on the top right of the layer list. 
+```
+
+## Save your data
+
+There are two options for saving your data: saving the layers themselves or saving the rendered view
+in the canvas. 
+
+To save the currently rendered view, use the File menu **Save Screenshot**. You can also chose to include
+the viewer or to save the screenshot to the clipboard instead of a file.
+
+You can save the layers individually using the File menu or Command+S(Mac) / Control+S(Windows). 
+* For image layers, you can select the file type by providing the extension. 
+Note: Some formats may not work for all image types, e.g. 3D stacks. Additional file types 
+can be made available by plugins.
+* Points and Shapes layers are saved as `.csv` files that contain their coordinates.
+* Labels layers are saved as `.tiff` files.
+
+Multiple different layer types, e.g. image and points, can be saved to a single `.svg` file,
+which can be convenient for use in a vector graphics program or for web. 
+* Select more than one layer and use the File menu or Command+S(Mac) / Control+S(Windows), 
+then in the dialog dropdown menu ensure `napari-svg` is selected, give the file a name, and save.
+
+    ```{important} 
+    The output file will not be openable in napari!
+    ```
+
+Perhaps more usefully, multiple different layer types, e.g. image and points, can be saved as
+individual layers (as described above) *to a folder.* 
+* Select more than one layer and use the File menu or Command+S(Mac) / Control+S(Windows) or use Command+Shift+S(Mac) / Control+Shift+S(Windows) to save all layers. Then in the dialog dropdown menu change `napari-svg`

```

Comment:
```suggestion
* Select more than one layer and use the File menu or Command+S (Mac) / Control+S (Windows and Linux) or use Command+Option+S (Mac) / Control+Alt+S (Windows and Linux) to save all layers. Then in the dialog dropdown menu change `napari-svg`

```

The Control+Shift+S shortcut did not work for me in Linux, neither did the Command+Shift+S on a Mac, so I tested and it's actually an Option/Alt shortcut (the menu lists the correct shortcut in both systems)